lib/load.php

```diff
@@ -151,6 +151,14 @@ function gutenberg_is_experiment_enabled( $name ) {
 require __DIR__ . '/demo.php';
 require __DIR__ . '/experiments-page.php';
 
+add_filter(
+	'safe_style_css',
+	function( $safe_rules ) {
+		$safe_rules[] = 'transition';
+		return $safe_rules;
+	}
+);
+
```

Comment:
⚠️ Obviously this is a temporary hack and will need to find a better home. Do we have a patch open for the Core function? Do we know _why_ it excludes `transition`?

Reply:
It's not in the allowed [attributes list](https://github.com/WordPress/wordpress-develop/blob/cab053271e272a21b4d140e50ac0d6a78094ecd5/src/wp-includes/kses.php#L2255).

There's no patch yet since this was all experimental to see if it floats.